airgun/browser.py

```diff
@@ -100,6 +70,7 @@ def __init__(
         """
         self.provider = provider or settings.selenium.browser
         self.browser = browser or settings.selenium.webdriver
+        self.web_kaifuku = yaml.load(settings.webkaifuku.config)
```

Comment:
Here is where I would look to consolidate the settings for airgun browser and webdriver in airgun.

Does airgun need a separate settings specification for browser and webdriver, with the same options repeated again in webkaifuku?  I don't think it does.

If we want to isolate this integration for airgun, I would not include a webkaifuku yaml block at all. Airgun maintains its own yaml spec for browser and webdriver settings. It then maps all of those settings to the kaifuku spec to call `BrowserManager.from_conf`.

If we're not as concerned about isolating the integration, then we consolidate on kaifuku yaml spec for browser and webdriver. Within airgun we read the relevant fields from this spec, instead of having a separate and unique yaml structure for the necessary options.

I think you should evaluate how `self.provider`, `self.browser`, and `settings.selenium` are used in airgun to help guide the decision of which of the above options best suits us.

I'm inclined to consolidate these settings on kaifuku's yaml spec.  Right now kaifuku isn't using any yaml validation tools, like dynaconf, to validate the structure of the dictionary passed. It's an enhancement for kaifuku as a library that we would want to discuss with Ronny.  

It's also a deeper enhancement for airgun - moving to using dynaconf for settings validation and specification, instead of ConfigParser. @latran @JacobCallahan

Maybe this is outside of the scope of kaifuku integration for now, but I would write it as an immediate issue/TODO against this settings repetition for webdriver and browser options.

Reply:
I also agree that airgun's settings should be moved over to dynaconf, but this is definitely outside the scope of this change.